tools/nl/embeddings/data/finetuning/sentence_pairs.csv

```diff
@@ -1,32 +1,118 @@
 sentence_1,sentence_2,score
-african americans,black people,1.0
-african americans,black americans,1.0
-african americans,african americans living in america,1.0
-african americans,black people in the US,1.0
-african americans,black and african american demographic,1.0
-african americans,population of black or african americans,1.0
+african americans,africa,0.2
+african americans,african americans living in america,1
+african americans,black americans,1
+african americans,black and african american demographic,1
+african americans,black people,1
+african americans,black people in the US,1
 african americans,foreign born,0.1
+african americans,group quarters,0.01
 african americans,immigrants,0.1
-african americans,africa,0.2
+african americans,incarcerated,0.01
 african americans,living in africa,0.2
 african americans,poor,0.01
-african americans,incarcerated,0.01
+african americans,population of black or african americans,1
 african americans,prison,0.01
+asian,inactivity,0.01
+asian,religion,0.01
+black,inactivity,0.01
+black,religion,0.01
+black men,poor,0.01
+black people,institutionalized,0.01
 black people,poor,0.01
 black people,prison,0.01
-hispanic women,poor,0.01
-hispanic women,prison,0.01
-hispanic women,incarcerated,0.01
+black population,group quarters,0.01
```

Comment:
For my understanding, is there any expectation of transitivity?

african americans vs. black people is 1
african americans vs. group quarters is 0.01

so, does that have any affect on black people vs. group quarters?

Reply:
There should be some transitivity, for sure. But I suspect it percolates a bit slowly so the more examples we can provide the more the transitive relationships will experience this too